pkgs/by-name/hu/hunspell/wrapper.nix

```
@@ -1,33 +1,39 @@
 {
-  stdenv,
   lib,
+  buildEnv,
   hunspell,
+  hunspellDicts,
   makeBinaryWrapper,
-  dicts ? [ ],
 }:
-let
-  searchPath = lib.makeSearchPath "share/hunspell" dicts;
-in
-stdenv.mkDerivation {
-  inherit (lib.appendToName "with-dicts" hunspell) name;
 
-  nativeBuildInputs = [ makeBinaryWrapper ];
+lib.makeOverridable (
+  { ... }@hunspellDicts:
```

It took me a minute to figure out that this `hunspellDicts` is not the one coming from `callPackage`. It may be a good idea, to name it something else

```suggestion
lib.makeOverridable (
  { ... }@dicts:
```